internal/client/job.go

```diff
@@ -60,12 +63,15 @@ func (c *Project) setupLocalJobSystem(ctx context.Context) (isLocal bool, newCtx
 	// Automatically determine if we should use a local or a remote runner
 	newCtx = ctx
 
+	// We may use this in multiple places, so we save the result if we obtain it
+	// NOTE(izaak): If in the future we need the full project in other places in this codepath, we should probably cache it early on the parent struct.
```

Comment:
Do you mind line-breaking this at anywhere from 70 to 100 columns? (usually I do 70). I know this is a hot topic but the rest of the project follows this style (and you do too in most places)

Reply:
oh ha, I don't mind at all. I'll update my editor to move the break to 70. Good to know I can cheat up to 100 and get away with it 😆.

Reply:
Do you go for 70 columns from where the comment starts, or 70 from the left margin?